spec/lib/sidekiq/form526_backup_submission_process/submit_spec.rb

```diff
@@ -84,13 +84,30 @@
                 expect(jid).to eq(jid_from_jobs)
                 described_class.drain
                 expect(jid).not_to be_empty
+
+                # After 526 succeeds ancillary form go through their backup submission process
```

Comment:
In the backup process, form 526 and all ancillary forms are gathered and submitted as an aggregate package. The processing of ancillary forms is not dependent on the success of form 526 submission as is the case in the primary path.